src/server/origin/summary/controller.js

```diff
@@ -0,0 +1,51 @@
+import { Origin } from '../../common/model/section/origin.js'
+const indexView = 'origin/summary/index.njk'
+export const pageTitle =
+  'Check your answers before you continue your application'
+export const heading = pageTitle
+
+/**
+ * @satisfies {Partial<ServerRoute>}
+ */
+export const originSummaryGetController = {
```

Comment:
I think this file is not needed as we are now using SummaryPageController

Reply:
removed - good catch :)